### PR TITLE
Replaced Creation Floating Action Buttons for Techniques

### PR DESCRIPTION
**Pull request by Bonsai8863**
### Proposed changes

* Replaced FAB for creation of Attack Patterns, Narratives, Courses of action, Data components, and Data sources.

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [x] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [x] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/techniques/Narratives.jsx [hunk 1]
```diff
@@ -15,11 +15,16 @@ import Breadcrumbs from '../../../components/Breadcrumbs';
...
+const ParametersHeader = styled('div')({
+  marginTop: -10,
+});
+
+const StyledSearchInput = styled(SearchInput)({
+  float: 'left',
+});
+
+const CreateNarrativeButton = styled('div')({
+  float: 'right',
```
Comment (labo-flg) [inline opencti-platform/opencti-front/src/private/components/techniques/Narratives.jsx]: Please use inline style instead

**Comment by labo-flg** [comment]
Same remarks as in https://github.com/OpenCTI-Platform/opencti/pull/7256.
A bit of code formatting + question about refactoring `CreateEntityControlledDial`

**Review by labo-flg — DISMISSED** [review]
Same remarks as in #7256
Once synchronized, I can do a quick testing and approve this one too.

File: opencti-platform/opencti-front/src/private/components/techniques/data_components/AddDataSources.tsx [hunk 2]
```diff
@@ -19,12 +20,16 @@ const useStyles = makeStyles(() => ({
     float: 'left',
     marginTop: -15,
   },
-  search: {
-    marginLeft: 'auto',
-    marginRight: ' 20px',
-  },
 }));
 
+const StyledSearchHeader = styled('div')({
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/techniques/data_components/AddDataSources.tsx]: Same here for styled

File: opencti-platform/opencti-front/src/private/components/techniques/data_components/DataComponentCreation.tsx [hunk 3]
```diff
@@ -36,6 +37,10 @@ import ProgressBar from '../../../../components/ProgressBar';
 import BulkTextField from '../../../../components/fields/BulkTextField/BulkTextField';
 import BulkTextModalButton from '../../../../components/fields/BulkTextField/BulkTextModalButton';
 
+const ContextualDataComponentCreationButton = styled('div')({
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/techniques/data_components/DataComponentCreation.tsx]: Same

File: opencti-platform/opencti-front/src/private/components/techniques/data_sources/AddDataComponents.tsx [hunk 4]
```diff
@@ -20,11 +21,16 @@ const useStyles = makeStyles(() => ({
     float: 'left',
     marginTop: -15,
   },
-  search: {
-    float: 'right',
-  },
 }));
 
+const StyledSearchHeader = styled('div')({
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/techniques/data_sources/AddDataComponents.tsx]: Same

File: opencti-platform/opencti-front/src/private/components/techniques/data_sources/DataSourceCreation.tsx [hunk 5]
```diff
@@ -38,6 +40,10 @@ import ProgressBar from '../../../../components/ProgressBar';
 import BulkTextField from '../../../../components/fields/BulkTextField/BulkTextField';
 import BulkTextModalButton from '../../../../components/fields/BulkTextField/BulkTextModalButton';
 
+const ContextualDataSourceCreateButton = styled('div')({
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/techniques/data_sources/DataSourceCreation.tsx]: Same

File: opencti-platform/opencti-front/src/private/components/techniques/narratives/AddSubNarrative.jsx [hunk 6]
```diff
@@ -16,10 +17,14 @@ const styles = () => ({
     float: 'left',
     marginTop: -15,
   },
-  search: {
-    marginLeft: 'auto',
-    marginRight: ' 20px',
-  },
+});
+
+const StyledSearchHeader = styled('div')({
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/techniques/narratives/AddSubNarrative.jsx]: Same

File: opencti-platform/opencti-front/src/private/components/techniques/narratives/NarrativeCreation.tsx [hunk 7]
```diff
@@ -37,6 +39,10 @@ import ProgressBar from '../../../../components/ProgressBar';
 import BulkTextField from '../../../../components/fields/BulkTextField/BulkTextField';
 import BulkTextModalButton from '../../../../components/fields/BulkTextField/BulkTextModalButton';
 
+const ContextualNarrativeCreateButton = styled('div')({
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/techniques/narratives/NarrativeCreation.tsx]: Same

File: opencti-platform/opencti-front/src/private/components/techniques/attack_patterns/AddSubAttackPattern.jsx [hunk 8]
```diff
@@ -57,6 +58,14 @@ const styles = (theme) => ({
   },
 });
 
+const StyledSearchHeader = styled('div')({
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/techniques/attack_patterns/AddSubAttackPattern.jsx]: To be replaced

File: opencti-platform/opencti-front/src/private/components/techniques/attack_patterns/AttackPatternCreation.tsx [hunk 9]
```diff
@@ -40,6 +44,21 @@ const useStyles = makeStyles<Theme>((theme) => ({
   },
 }));
 
+const ContextualStyle = styled('div')(({ display }: { display?: boolean }) => ({
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/techniques/attack_patterns/AttackPatternCreation.tsx]: Same

**Review by Kedae — APPROVED** [review]

**Pull request merged**